nbuild/00.Foreword.md

```diff
@@ -0,0 +1,17 @@
+# Foreword
+Welcome to Nest build's official getting-started guide!
+
+Nest build, or `nbuild`, is an automated package builder for [Raven-OS](https://raven-os.org), which is an adaptative, customizable and evolutive open-source Operating System (OS) based on the Linux kernel.
+`nbuild` can be run on any linux-based distribution.
+
+## Documentation
+`nbuild`'s documentation is hosted [there](https://docs.raven-os.org/nbuild/master), but it can also be locally generated with the following commands:
```

Comment:
```suggestion
`nbuild`'s documentation is hosted [here](https://docs.raven-os.org/nbuild/master), but it can also be locally generated with the following commands:
```